src/pages/rechner/[frage].tsx

```diff
@@ -139,4 +142,17 @@ export const getStaticProps: GetStaticProps<Props> = async ({ params }) => {
   };
```

Comment:
You should probably add a `revalidate` prop so that you don't have to redeploy the project everytime you change some question in firestore.